### DISCUSSION TITLE
Denoise video and low light shoot

### DISCUSSION THREAD

**Discussion by And1re** [gopro/labs#755] [Q&A] [answered]
What settings do I need to make in GoPro labs via QR code to remove video noise and graininess, as well as shoot in low light? 

Please Help 😞

**Comment by dnewman-gpsw** [comment]
That's somewhat the opposite direction, Labs allows you to lessen the noise reduction, which means more graininess, but significantly more detail, and that allows you to get better results using post tools, for more information in low light. Post tools like NeatVideo and DaVinci Resolve are the favorites for denoising. There are other extensions to control exposure better for some lower light conditions, but that not what you asked.  There is a lot to learn.

**Reply by And1re** [reply]
Thank you very much 🙏🏻
And what settings I need to make in labs to more light in dark scenes ?

**Reply by dnewman-gpsw** [reply, marked as answer]
That is almost like asking how exposure works in photography.  You need a longer shutter, or more ISO, your choice.  This is not single solution, always with tradeoffs.

**Reply by And1re** [reply]
Thank you so much 🙏🏻
